apps/testnorge-arena/arena-core/src/main/java/no/nav/registre/arena/core/consumer/rs/request/RettighetEndreDeltakerstatusRequest.java

```diff
@@ -0,0 +1,40 @@
+package no.nav.registre.arena.core.consumer.rs.request;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import no.nav.registre.testnorge.domain.dto.arena.testnorge.vedtak.NyttVedtakAap;
+import no.nav.registre.testnorge.domain.dto.arena.testnorge.vedtak.NyttVedtakTillegg;
+import no.nav.registre.testnorge.domain.dto.arena.testnorge.vedtak.NyttVedtakTiltak;
+import no.nav.registre.testnorge.domain.dto.arena.testnorge.vedtak.NyttVedtakEndreDeltakerstatus;
+
+import java.util.Collections;
+import java.util.List;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class RettighetEndreDeltakerstatusRequest extends RettighetRequest {
+
+    private List<NyttVedtakEndreDeltakerstatus> nyeEndreDeltakerstatus;
```

Comment:
Se kommentarene mine på NyttVedtakResponse. Hvis du går for å legge de nye feltene inn i NyttVedtakTiltak gjelder dette:

Typen her kan eventuelt endres til NyttVedtakTiltak, og så legger du til en @JsonProperty("nyeEndreDeltakerstatus") på metoden getVedtakTiltak i denne klassen, og returnerer nyeEndreDeltakerstatus. 

Så det blir:

```
public class RettighetEndreDeltakerstatusRequest extends RettighetRequest {

    private List<NyttVedtakTiltak> nyeEndreDeltakerstatus;

    @Override public String getArenaForvalterUrlPath() {
        return "/v1/endreDeltakerstatus";
    }

    @Override public List<NyttVedtakAap> getVedtakAap() {
        return Collections.emptyList();
    }

    @JsonProperty("nyeEndreDeltakerstatus")
    @Override public List<NyttVedtakTiltak> getVedtakTiltak() {
        return nyeEndreDeltakerstatus;
    }

    @Override public List<NyttVedtakTillegg> getVedtakTillegg() {
        return Collections.emptyList();
    }
}
```

I tillegg må du inn i klassen RettighetRequest og legge til en ny `@JsonSubTypes.Type(value = RettighetEndreDeltakerstatusRequest.class, name = "endreDeltakerstatus")` for å få riktig json-serialisering/deserialisering

Reply:
Fikset :)